lisa/wa_results_collector.py

```diff
@@ -492,6 +492,48 @@ def row_filter(r): return True
 
         return ret
 
+    def _read_energy_instrument_metrics(self, path):
+        """
+        Parse the WA energy_measurement output.
+        """
+        path_cache = path + ".parquet"
+
+        if not os.path.isfile(path_cache):
```

Comment:
Could swap both legs and remove the negation in the condition

Reply:
Done